app/src/main/java/fr/free/nrw/commons/nearby/mvp/contract/NearbyParentFragmentContract.java

```diff
@@ -0,0 +1,58 @@
+package fr.free.nrw.commons.nearby.mvp.contract;
+
+import android.view.View;
+
+import com.mapbox.mapboxsdk.annotations.Marker;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+import java.util.List;
+
+import fr.free.nrw.commons.kvstore.JsonKvStore;
+import fr.free.nrw.commons.location.LatLng;
+import fr.free.nrw.commons.location.LocationServiceManager;
+import fr.free.nrw.commons.nearby.Place;
+
+public interface NearbyParentFragmentContract {
+
+    interface View {
+        void registerLocationUpdates(LocationServiceManager locationServiceManager);
+        boolean isNetworkConnectionEstablished();
+        void addNetworkBroadcastReceiver();
+        void listOptionMenuItemClicked();
+        void  populatePlaces(LatLng curlatLng, LatLng searchLatLng);
+        boolean isListBottomSheetExpanded();
+        void checkPermissionsAndPerformAction(Runnable runnable);
+        void displayLoginSkippedWarning();
+        void setFABPlusAction(android.view.View.OnClickListener onClickListener);
+        void setFABRecenterAction(android.view.View.OnClickListener onClickListener);
+        void animateFABs();
+        void recenterMap(LatLng curLatLng);
+        void hideBottomSheet();
+        void displayBottomSheetWithInfo(Marker marker);
+        void addOnCameraMoveListener(MapboxMap.OnCameraMoveListener onCameraMoveListener);
+        void addSearchThisAreaButtonAction();
+        void setSearchThisAreaButtonVisibility(boolean isVisible);
+        void setProgressBarVisibility(boolean isVisible);
+        void setTabItemContributions();
+        boolean isDetailsBottomSheetVisible();
+        void setBottomSheetDetailsSmaller();
+    }
+
+    interface ListView {
```

Comment:
Lets rename this to something else, ListView is android framework keyword

Reply:
I agree to this